src/main/java/edu/nps/moves/dis/ElectronicEmissionSystemData.java

```diff
@@ -69,14 +69,20 @@ public int getMarshalledSize() {
         return marshalSize;
     }
 
-    public void setSystemDataLength(short pSystemDataLength) {
-        systemDataLength = pSystemDataLength;
-    }
-
     public short getSystemDataLength() {
         return systemDataLength;
     }
 
+    private short calculateSystemDataLength() {
+        //systemdata length = 13*beamCount + SUM(beamsCount)2*track/jamTargets
+        short size = 5;
```

Comment:
What's the size = 5 represent here?

Reply:
The 5 represents 32-bit words for the fields in the pdu shown in  figure  below, and is a part of the **System Data Length.** 

> _**System Data Length.** This field shall specify the length of this emitter system’s data (including
> beam data and its track/jam information) in 32-bit words. The length shall include the System
> Data Length field. This field shall be represented by an 8-bit unsigned integer._
> 

![system_data_length](https://github.com/open-dis/open-dis-java/assets/92848575/3f9c2cf4-ce95-422b-b9a4-fd8d834b079a)